### PR TITLE
Fix: Daylight savings timezone offset for reporting year seed dates

### PR DESCRIPTION
**Pull request by kriscooke**
In the course of #565 @matthieu-foucault brought to light that the timezone offsets of our seed dates for reporting years are off by one (winter dates should be -8 instead of -7).

### CODE REVIEW

**Comment by dleard** [comment]
@kriscooke @matthieu-foucault does this need to account for BC moving away from daylight savings after this year?

**Comment by kriscooke** [comment]
This is just for the pre-populated seed years, and there was no concrete decision timeline for that change. BC will get rid of DST "whenever California and Washington _et al._ do".

We'll just have to update it later if and when that change happens.
https://engage.gov.bc.ca/govtogetherbc/impact/daylight-savings-results/

File: schema/data/prod/reporting_year.sql [hunk 1]
```diff
@@ -4,11 +4,11 @@ with rows as (
 insert into ggircs_portal.reporting_year(reporting_year, reporting_period_start, reporting_period_end, application_open_time, application_close_time, application_response_time)
 overriding system value
 values
-(2018, '2018-01-01 00:00:00.0-07', '2018-12-31 23:59:59.0-07', '2019-04-01 14:49:54.191757-07', '2019-12-30 14:49:54.191757-07', '2020-01-30 14:49:54.191757-07'),
-(2019, '2019-01-01 00:00:00.0-07', '2019-12-31 23:59:59.0-07', '2020-04-01 14:49:54.191757-07', '2020-12-30 14:49:54.191757-07', '2021-01-30 14:49:54.191757-07'),
-(2020, '2020-01-01 00:00:00.0-07', '2020-12-31 23:59:59.0-07', '2021-04-01 14:49:54.191757-07', '2021-12-30 14:49:54.191757-07', '2022-01-30 14:49:54.191757-07'),
-(2021, '2021-01-01 00:00:00.0-07', '2021-12-31 23:59:59.0-07', '2022-04-01 14:49:54.191757-07', '2022-12-30 14:49:54.191757-07', '2023-01-30 14:49:54.191757-07'),
-(2022, '2022-01-01 00:00:00.0-07', '2022-12-31 23:59:59.0-07', '2023-04-01 14:49:54.191757-07', '2023-12-30 14:49:54.191757-07', '2024-01-30 14:49:54.191757-07')
+(2018, '2018-01-01 00:00:00.0-08', '2018-12-31 23:59:59.0-08', '2019-04-01 14:49:54.191757-07', '2019-12-30 14:49:54.191757-07', '2020-01-30 14:49:54.191757-07'),
```
Comment (dleard) [inline schema/data/prod/reporting_year.sql:7]: Don't the application_close_time & application_response_time dates fall into the winter as well?
```suggestion
(2018, '2018-01-01 00:00:00.0-08', '2018-12-31 23:59:59.0-08', '2019-04-01 14:49:54.191757-07', '2019-12-30 14:49:54.191757-08', '2020-01-30 14:49:54.191757-08'),
```

Comment (kriscooke) [reply schema/data/prod/reporting_year.sql:7]: Yes, thank you!

**Pull request merged**